podpac/core/coordinates/coordinates.py

```diff
@@ -19,6 +19,10 @@
 from six import string_types
 import pyproj
 import logging
+from scipy import spatial
+from geopy.distance import geodesic

```

Comment:
Whenever you add a new package, you MUST add it to the setup.py file. Otherwise I can't run the code. I'll make this fix. 

Also, when it's an optional dependency, it's better to lazy-load the function or module. In this case I think the dependency should be part of the core functionality.